lldb/source/Plugins/ExpressionParser/Swift/SwiftExpressionParser.cpp

```diff
@@ -1162,7 +1163,30 @@ static llvm::Expected<ParsedExpression> ParseAndImport(
   if (playground) {
     auto *persistent_state =
         sc.target_sp->GetSwiftPersistentExpressionState(exe_scope);
-    persistent_state->AddHandLoadedModule(ConstString("Swift"));
+
+    lldb::StackFrameSP this_frame_sp(stack_frame_wp.lock());
+    if (!this_frame_sp)
+      return make_error<StringError>(llvm::inconvertibleErrorCode(),
+                                     "[SEP:PAI] No stackframe!");
```

Comment:
This might be done for consistency but generally errors should not start with capitals or end with punctuation (because they're commonly appended: error: error1: error2.